### PR TITLE
Fixing Integration Workflow Rule Break

### PR DESCRIPTION
**Pull request by kaihokori**
# Description

## Summary of Changes

- The format job has been updated to not run when the commit is to main. 

## Additional Comments

The main branch is protected and requires a pull request to be made. The format job in the continuous integration workflow mimics the last user and attempts to push a new commit to whatever branch the last commit was on. When a pull request is merged into main, the format jobs does its thing and attempts to push changes to main but since it's mimicking a real user, it gets an error. 

Since the linting job is set to need the formatting job, the continuous integration as a whole will not operate on the main branch when a PR is merged.

### CODE REVIEW

**Comment by kaihokori** [comment]
Accidentally branched off of #9. Please don't merge this PR until #9 is merged.

**Review by NhanDotJS — APPROVED** [review]

**Pull request merged**
